### DISCUSSION TITLE
Can't set description of repository until commits are added

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#12510] [Other Feature Feedback, Questions, & Ideas]
The right side of https://github.com/github/feedback/ has:

<img width="320" alt="image" src="https://user-images.githubusercontent.com/2119212/157062524-6cddb163-4a5c-40c4-9067-478d0a54e4a1.png">

> Public feedback discussions for: GitHub for Mobile, GitHub Discussions, GitHub Codespaces, GitHub Sponsors, GitHub Issues and more!

Unfortunately, that comes at a steep cost, spammy comments such as those in:
<img width="924" alt="https://github.com/github/feedback/pull/12296#pullrequestreview-899768982" src="https://user-images.githubusercontent.com/2119212/157062328-ad2b13b5-12f2-42b0-b191-ef6502ad3e8f.png">

I've created a repository https://github.com/jsoref/dark-wiki whose only purpose is to host a wiki (specifically to provide feedback on quirky wiki behavior).

I'd like to set the description of the repository so that users can see it instead of just seeing:
<img width="411" alt="This repository is empty.
Care to check out the [GitHub Channel on YouTube](https://youtube.com/GitHub) while you wait?" src="https://user-images.githubusercontent.com/2119212/157062093-554bf685-ce28-400d-9d06-7ce5d93e07aa.png">

Apparently my repository already has a ⭐ , I'm not quite sure _why_, but normal people won't be able to see that because the sidebar is missing. And I'd hope that by being able to give the repository a description, maybe I could prevent people from needlessly 🌟 ing it.

**Comment by eloekset** [comment]
When a new repo is created, you're prompted for name and description. The settings page has a text box for renaming the repo, but no field for change the description. That's the place were description editing should be.
